pkg/dataplane/cert.go

```diff
@@ -146,6 +146,7 @@ func EnsureTLSCerts(ctx context.Context, helper *helper.Helper,
 				}
 			}
 		}
+		sort.Strings(ips)
```

Comment:
note: this is needed so that we don't keep updating the Certificate with a different list and causing a reconcile loop.